plasmapy/analysis/swept_langmuir/floating_potential.py

```diff
@@ -297,4 +297,4 @@ def find_floating_potential(
 
 
 find_vf_ = find_floating_potential
-"""Alias to :func:`find_floating_potential`. """
+"""Alias to :func:`find_floating_potential`."""
```

Comment:
Sorry I missed this...it appears I was spacing out.  😬

Reply:
Looks like we found it independently at about roughly the same time. 🤣